pom.xml

```diff
@@ -73,9 +73,9 @@
     </dependency>
 
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.13.2</version>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>5.8.0</version>
```

Comment:
Ok, now it will be next major version.

so we can put junit in test scope.

and remove all deprecated method from production code.

Reply:
https://issues.apache.org/jira/browse/MSHARED-1043